website/docs/commands/misc/config.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 
 import {ChainedSnippets} from "../../../src/components/MarkdownComponents.js";
 
-The `config` sub-command allows to get and set various configuration values, used by
+The `config` sub-command makes it possible to get and set various configuration values, used by
```

Comment:
I think the unchanged version of this line is better, actually (and both are grammatically correct)